.github/mergify.yml

```diff
@@ -0,0 +1,22 @@
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=all-ci
```

Comment:
This "all-ci" is the name of the CI pipeline which has to succeed for the PR to be mergeable